tasks/collect-mrrc-params/tests/test-collect-mrrc-params.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-collect-mrrc-params
+spec:
+  description: |
+    Run the collect-mrrc-params task
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > "$(workspaces.data.path)/data.json" << EOF
+              {
+                "mrrc": {
+                  "product": {
+                    "name": "activemq",
+                    "version": "0.0.1",
+                    "cosignPubKeyConfig": "test-cosign-pub-key",
+                    "zipRegistry": "quay.io/ligangty/activemq.zip:0.0.1"
+                  },
+                  "awsSecret": "charon-aws-credentials",
+                  "environment": "dev",
+                  "release": "ga"
+                }
+              }
+              EOF
+    - name: run-task
+      taskRef:
+        name: collect-mrrc-params
+      params:
+        - name: dataJsonPath
+          value: data.json
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      params:
+        - name: mrrcParamFilePath
+          value: $(tasks.run-task.results.mrrcParamFilePath)
+        - name: cosignPubKeyConfig
+          value: $(tasks.run-task.results.cosignPubKeyConfig)
+        - name: charonAWSSecret
+          value: $(tasks.run-task.results.charonAWSSecret)
+      taskSpec:
```

Comment:
Can you try adding the params here? See https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/collect-pyxis-params/tests/test-collect-pyxis-params.yaml#L52

Reply:
Added